src/configuration/augment_config.coffee

```diff
@@ -28,3 +28,4 @@ module.exports = (config) ->
     config.docDirective[name] = value.renderedAttr
     config.templateAttrLookup[value.attr] = name
 
+  config
```

Comment:
It seems like I disabled it manually. Donno why and how this happened. Anyway, `ensure_newline_at_eof_on_save: true` is now set. Sorry about that one.